hip_data_tools/aws/s3.py

```diff
@@ -102,6 +102,11 @@ def upload_dataframe_as_parquet(self,
             file_name (str): the name of the file at destination
         Returns: None
         """
+        log.debug("Uploading the dataframe as parquet")
+        log.debug("Column names of the dataframe: %s", list(dataframe))
```

Comment:
add to the same log statement.

Reply:
if you have multiple statements, they are prone to get diluted by other logs

Reply:
fixed